plugins/qtosgrave/qtosgviewer.cpp

```diff
@@ -180,6 +181,14 @@ QtOSGViewer::QtOSGViewer(EnvironmentBasePtr penv, std::istream& sinput) : QMainW
     _InitGUI(bCreateStatusBar, bCreateMenu);
     _bUpdateEnvironment = true;
     _bExternalLoop = false;
+
+    // Read copy QT resource to temp location and later stream that into OSG to use when making labels
+    QFile fontFile(":/fonts/NotoSans-Regular.ttf");
```

Comment:
@Hsifnus what if `:/fonts/NotoSans-Regular.ttf` does not exist? is there a backup?

Reply:
It is integrated into `QResource` and we ship that.

Reply:
https://github.com/rdiankov/openrave/blob/7cd038f9b07264920b4f10a95127553d7e247618/plugins/qtosgrave/qtosgviewer.qrc#L3